src/nbl/video/COpenGLPhysicalDevice.h

```diff
@@ -32,15 +32,16 @@ class COpenGLPhysicalDevice final : public IOpenGL_PhysicalDeviceBase<COpenGLLog
 protected:
 	core::smart_refctd_ptr<ILogicalDevice> createLogicalDevice_impl(const ILogicalDevice::SCreationParams& params) final override
 	{
-		return core::make_smart_refctd_ptr<COpenGLLogicalDevice>(m_egl, &m_glfeatures, m_config, m_gl_major, m_gl_minor, params, m_dbgCb, core::smart_refctd_ptr(m_system), core::smart_refctd_ptr(m_GLSLCompiler));
+		return core::make_smart_refctd_ptr<COpenGLLogicalDevice>(m_egl, &m_glfeatures, m_config, m_gl_major, m_gl_minor, params, m_dbgCb, core::smart_refctd_ptr(m_system), core::smart_refctd_ptr(m_GLSLCompiler), core::smart_refctd_ptr(m_logger));
 	}
 
 private:
-    COpenGLPhysicalDevice(core::smart_refctd_ptr<system::ISystem>&& s, core::smart_refctd_ptr<asset::IGLSLCompiler>&& glslc, const egl::CEGL* _egl, EGLConfig config, EGLContext ctx, EGLint major, EGLint minor, SDebugCallback* dbgCb) :
-        base_t(std::move(s), std::move(glslc), _egl, config, ctx, major, minor, dbgCb)
+    COpenGLPhysicalDevice(core::smart_refctd_ptr<system::ISystem>&& s, core::smart_refctd_ptr<asset::IGLSLCompiler>&& glslc, const egl::CEGL* _egl, EGLConfig config, EGLContext ctx, EGLint major, EGLint minor, SDebugCallback* dbgCb, core::smart_refctd_ptr<system::ILogger>&& logger) :
+        base_t(std::move(s), std::move(glslc), _egl, config, ctx, major, minor, dbgCb, std::move(logger))
     {
 
     }
+	_NBL_DEBUG
```

Comment:
huh?

Reply:
oops, the side effect of doing non-stop manual find&replace XD